students/alexander_boone/lesson07/html_render.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+
+"""
+A class-based system for rendering html.
+"""
+
+
+# This is the framework for the base class
+class Element(object):
+
+    tag = "html"
+    indent = "    "
+
+    def __init__(self, content=None, **kwargs):
+        self.attributes = dict()
+        if content is None:
+            self.contents = []
+        else:
+            self.contents = [content]
+        for key, value in kwargs.items():
+            self.attributes[key] = value
```

Comment:
why not just assign kwargs to self.attributes? are you attempting a deep copy? if so, could you use a comprehension here? you can also use copy.deepcopy which makes the intent more clear